apps/api/src/domain/models/playlist/entities/playlist.entity.ts

```diff
@@ -109,4 +123,53 @@ export class Playlist extends Resource {
     protected getExternalReferences(): AggregateCompositeIdentifier<AggregateType>[] {
         return this.items.map(({ resourceCompositeIdentifier }) => resourceCompositeIdentifier);
     }
+
+    handlePlaylistNameTranslated({
+        payload: { text, languageCode },
+    }: PlaylistNameTranslated): ResultOrError<Playlist> {
+        return this.translateName(
```

Comment:
We should change the API of `translateName` so that you don't pass in the `MultilingualTextItem` but just the props or a DTO. I don't think the `role` should be passed in, but rather hard-wired. Let's make a separate story for this.